https://discourse.onlinedegree.iitm.ac.in/t/164277

Also is it necessary to pass the email as argument?</p><hr>

<h4>23f2003413 (#423)</h4>
<p><div class="lightbox-wrapper"><a class="lightbox" data-download-href="/uploads/short-url/uzReAwAPsgAOBKSrr4jwARzOkLg.png?dl=1" href="https://europe1.discourse-cdn.com/flex013/uploads/iitm/original/3X/d/6/d64ee81798b48ccad186d5051823d3f565424bc2.png" rel="noopener nofollow ugc" title="image"><div class="meta"><svg aria-hidden="true" class="fa d-icon d-icon-far-image svg-icon"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1831×146 7.91 KB</span><svg aria-hidden="true" class="fa d-icon d-icon-discourse-expand svg-icon"><use href="#discourse-expand"></use></svg></div></a></div><br/>
can someone help me fix this error <a class="mention" href="/u/jivraj">@Jivraj</a> <a class="mention" href="/u/carlton">@carlton</a></p><hr>

<h4>Sagan (#422)</h4>
<p>Is it import to use python 3.13?<br/>
It is not stable yet</p><hr>

<h4>22f3002723 (#421)</h4>
<p>please ignore the above… there was a upper case issue  in image name… now fine</p><hr>

<h4>22f3002723 (#420)</h4>
<p>facing the issue on submission<br/>
<div class="lightbox-wrapper"><a class="lightbox" data-download-href="/uploads/short-url/jEwu0RbXwJSFn5jXec9n8gXzJWW.png?dl=1" href="https://europe1.discourse-cdn.com/flex013/uploads/iitm/original/3X/8/9/89bdffb424290fa15cf3f07c367b81fac5898b12.png" rel="noopener nofollow ugc" title="image"><div class="meta"><svg aria-hidden="true" class="fa d-icon d-icon-far-image svg-icon"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">942×521 28.7 KB</span><svg aria-hidden="true" class="fa d-icon d-icon-discourse-expand svg-icon"><use href="#discourse-expand"></use></svg></div></a></div></p><hr>

<h4>AyushTiwari (#460)</h4>
<p>Please extend deadline by 1 day.